lib/web/mage/sticky.js

```diff
@@ -11,7 +11,26 @@ define([
 
     $.widget('mage.sticky', {
         options: {
-            container: ''
+            container: '',
+            spacingTop: 0,
+            stickAfter: 0,
+            stickyClass: '_sticky'
+        },
+
+        /**
+         * Retrieve option value
+         * @param  {String} option
+         * @return {*}
+         * @private
+         */
+        _getOptionValue: function (option) {
+            var value = this.options[option] || 0;
+
+            if (typeof value === 'function') {
```

Comment:
It could be useful to set a function for `stickAfter` parameter that will calculate sticky element height when you wish to stick element after it goes out of the viewport and the element could change its height dynamically (shopping cart for example)
You can use different spacing top value for different screen size/orientation also.